src/main/java/com/google/ehub/servlets/ItemSubmissionServlet.java

```diff
@@ -45,13 +67,24 @@ public void doPost(HttpServletRequest request, HttpServletResponse response) thr
    * @param title the title given in the Post request parameter
    * @param description the description given in the Post request parameter
    * @param imageUrl the image URL given in the Post request parameter
+   * @param runtime the runtime given in the Post request parameter
+   * @param genre the genre given in the Post request parameter
+   * @param directors the directors given in the Post request parameter
+   * @param writers the writers given in the Post request parameter
+   * @param actors the actors given in the Post request parameter
    * @return true if the parameters given in the Post request are valid, false otherwise
    */
-  private static boolean arePostRequestParametersValid(
-      String title, String description, String imageUrl) {
+  private static boolean arePostRequestParametersValid(String title, String description,
+      String imageUrl, String releaseDate, String runtime, String genre, String directors,
+      String writers, String actors) {
     return (title != null && !title.isEmpty() && title.length() <= MAX_TITLE_CHARS)
-        && (description != null && !description.isEmpty()
-            && description.length() <= MAX_DESCRIPTION_CHARS)
-        && (imageUrl != null && !imageUrl.isEmpty());
+        && (description != null && !description.isEmpty() && description.length() <= MAX_CHARS)
```

Comment:
Since you apply the same 3 checks to each string, you can move it to a helper function

Reply:
I added a helper function to avoid code duplication in ItemSubmissionServlet in last commit.